docs/sql-ref-syntax-qry-select-hints.md

```diff
@@ -126,6 +126,10 @@ Join hints allow users to suggest the join strategy that Spark should use. Prior
 
     Suggests that Spark use shuffle-and-replicate nested loop join.
 
+* **BROADCAST_NL**
+
+    Suggests that Spark use broadcast nested loop join. The join side with the hint will be broadcast regardless of `autoBroadcastJoinThreshold`. If both sides of the join have the broadcast hints, the one with the smaller size (based on stats) will be broadcast. Note that, this hint is only useful if the join type and build side is not supported by hash join. i.e. full outer join, left outer/semi/anti join with build left, right outer join with build right.
+
```

Comment:
`hash join` means broadcast or shuffle hash join?

Reply:
it should be broadcast hash join, let me make it clear